README.md

```diff
@@ -73,7 +73,7 @@ This repository contains a `.repos` file that helps you clone the required depen
     ```
 4. **Install the dependencies**
     ```bash
-    vcs import src < src/franka.repos --recursive --skip-existing
+    vcs import src < src/franka_ros2/franka.repos --recursive --skip-existing
```

Comment:
Actuall this is wrong cause we are cloning the franka_ros2 in the upper step like this
git clone https://github.com/frankaemika/franka_ros2.git src
this will rename the franka_ros2 src. So there is no franka_ros2 in between anymore